index.html

```diff
@@ -195,6 +196,9 @@ <h2 class="h2">Get Our Newsletter</h2>
           <a target="_blank" href="https://www.linkedin.com/company/19037531/">
             <img src="/assets/icons/linked-in-mark.svg" alt="LinkedIn Icon" />
           </a>
+          <a target="_blank" href="https://www.youtube.com/watch?v=DLzxrzFCyOs&ab_channel=AllKindsOfStuff">
+              <img src="/assets/icons/free-youtube-icon-123-thumb.svg" alt="YouTube Icon" style="height: 25px; width: 25px;"/>

```

Comment:
maybe extract youtube styling since its replicated

Reply:
to classname / stylesheet

Reply:
updated the .svg image itself to the proper size